cmake/SetupBuildStageTargets.cmake

```diff
@@ -83,6 +83,7 @@ add_dependencies(test-libs-pointwise-functions
   Test_GrMhdAnalyticData
   Test_BurgersSolutions
   Test_ElasticitySolutions
+  Test_EquationsOfState
```

Comment:
@kidder is this the change you mean? It does show up for me..  so perhaps it's GitHub acting weird?

Reply:
yes, I see it now...